packages/cli/test/helpers/setup-fixture.ts

```diff
@@ -46,6 +46,23 @@ export function setupFixture(fixtureName: string) {
   return cwd;
 }
 
+export function setupFixtureFromTemplate(
+  fixture: string,
+  template: string,
+  nestedPaths?: string[]
+) {
+  const cwd = setupFixture(fixture);
+  const templateFixture = path.join(fixturesRoot!, 'unit', template);
```

Comment:
**suggestion (soft-blocking):** ensure `fixturesRoot` is populated

Are we sure that `fixturesRoot` is populated here? It looks like it's only populated if `setupFixture` has been called, but if `packages/cli/test/unit/util/input/project-selector.test.ts` is run alone, that never happens.

It seems like this would work in a full run, but fail if you isolate and run this test file.

In this or a follow-up PR: Can we ensure that `fixturesRoot` is populated here?